atomic_scanners/scanner-rpm-verify/rpm_verify.py

```diff
@@ -196,6 +197,11 @@ def run(self):
         # print error
         if not result:
             result = ["No issues. Libraries and Binaries are intact."]
+            self.json_out["Summary"] = "Libraries and binaries in the image"\
+                " are intact."
+        else:
+            self.json_out["Summary"] = "rpm -Va reported issues with some "\
```

Comment:
can you change this to
"RPM verify scanner reported [..]"

Reply:
Done